src/flamegraph/mod.rs

```diff
@@ -254,15 +270,37 @@ where
             write!(buffer, "all ({} samples, 100%)", samples_txt)
         } else {
             let pct = (100 * samples) as f64 / timemax as f64;
-
-            // strip any annotation
-            write!(
-                buffer,
-                "{} ({} samples, {:.2}%)",
-                deannotate(&frame.location.function),
-                samples_txt,
-                pct
-            )
+            match frame.delta {
+                None => write!(
+                    buffer,
+                    "{} ({} samples, {:.2}%)",
+                    deannotate(&frame.location.function),
+                    samples_txt,
+                    pct
+                ),
+                // Special case delta == 0 so we don't format percentage with a + sign.
```

Comment:
Bah, that's annoying. But seems reasonable.

Reply:
I wish there was a way to format with + and -, except when 0.